README.rst

```diff
@@ -35,32 +39,59 @@ Next, configure your aws credentials and default region:
     Default region name [us-east-1]:
     Default output format [None]:
 
-Then, run pcluster configure:
+Then, run ``pcluster configure``. A list of valid options will be displayed for each
+configuration parameter: type an option number and press ``Enter`` to confirm your choice, or just press ``Enter``
+to accept the default option.
 
 .. code-block:: ini
 
   $ pcluster configure
-  Cluster Template [default]:
-  Acceptable Values for AWS Region ID:
-      ap-south-1
-      ...
-      us-west-2
+  INFO: Configuration file /tmp/test will be written.
+  Press CTRL-C to interrupt the procedure.
+
+
+  Allowed values for AWS Region ID:
+  1. eu-north-1
+  ...
+  15. us-west-1
+  16. us-west-2
   AWS Region ID [us-east-1]:
-  VPC Name [myvpc]:
-  Acceptable Values for Key Name:
-    keypair1
-    keypair-test
-    production-key
-  Key Name []:
-  Acceptable Values for VPC ID:
-    vpc-1kd24879
-    vpc-blk4982d
-  VPC ID []:
-  Acceptable Values for Master Subnet ID:
-    subnet-9k284a6f
-    subnet-1k01g357
-    subnet-b921nv04
-  Master Subnet ID []:
+  ...
+
+Make sure to select a region where you own at least a key pair otherwise the process will end with this message.
```

Comment:
where -> on which ?

Then I'd remove the error message, because in future we can change the error message and this help will became outdated.

I'd write: "Make sure to select a region on which you own at least an EC2 key pair."

Reply:
Done